pyproject.toml

```diff
@@ -32,6 +32,7 @@ filterwarnings = [
     'ignore:`resume_download` is deprecated and will be removed in version 1.0.0. Downloads always resume when possible. If you want to force a new download, use `force_download=True`.',  # transformers calls deprecated hf_hub
     "ignore:`torch.cuda.amp.GradScaler",  # GradScaler changes in torch 2.3.0 but we want to be backwards compatible.
     "ignore:`clean_up_tokenization_spaces` was not set",  # Default behavior changes in transformers v4.45, raising irrelevant FutureWarning for serialized models.
+    "ignore:1Torch was not compiled with flash attention", # You might want to install flash attention, but you don't have to.
```

Comment:
Is this a typo?

Reply:
Oops, sorry, I've removed it. Originally I copied it in from helpmefindaname's [commit](https://github.com/flairNLP/flair/pull/3560/commits/783ebc7c5db43f2d0e3ca0d6473556da34a4e70e) fixing CI, but good catch, it's unnecessary for this PR.